control-plane-operator/controllers/hostedcontrolplane/pki/kas.go

```diff
@@ -40,10 +45,14 @@ func ReconcileKASServerCertSecret(secret, ca *corev1.Secret, ownerRef config.Own
 		fmt.Sprintf("%s.%s.svc", svc.Name, svc.Namespace),
 		fmt.Sprintf("%s.%s.svc.cluster.local", svc.Name, svc.Namespace),
 	}
+	dnsNames = append(dnsNames, svcAddresses...)
```

Comment:
if svcAddresses is a slice of IP's, why add it to dns names?

Reply:
Because this way is how was set in the Mario's env.

OCP Original:
```
DNS:kubernetes, DNS:kubernetes.default, DNS:kubernetes.default.svc, DNS:kubernetes.default.svc.cluster.local, DNS:openshift, DNS:openshift.default, DNS:openshift.default.svc, DNS:openshift.default.svc.cluster.local, DNS:172.30.0.1, DNS:fd02::1, IP Address:172.30.0.1, IP Address:FD02:0:0:0:0:0:0:1
```

Env:
```
DNS:kubernetes, DNS:kubernetes.default, DNS:kubernetes.default.svc, DNS:kubernetes.default.svc.cluster.local, DNS:openshift, DNS:openshift.default, DNS:openshift.default.svc, DNS:openshift.default.svc.cluster.local, DNS:172.30.0.1, DNS:fd02::1, IP Address:172.30.0.1, IP Address:FD02:0:0:0:0:0:0:1
```

New:
```
DNS:localhost, DNS:kubernetes, DNS:kubernetes.default, DNS:kubernetes.default.svc, DNS:kubernetes.default.svc.cluster.local, DNS:kube-apiserver, DNS:kube-apiserver.clusters-hosted.svc, DNS:kube-apiserver.clusters-hosted.svc.cluster.local, DNS:api.hosted.dual.lab, DNS:api.hosted.hypershift.local, DNS:172.31.0.1, DNS:fd02::1, IP Address:127.0.0.1, IP Address:0:0:0:0:0:0:0:1, IP Address:172.31.0.1, IP Address:FD02:0:0:0:0:0:0:1
```